strings/tips.txt

```diff
@@ -19,14 +19,15 @@ Maintenance is full of equipment that is randomized every round. Look around and
 Some roles cannot be antagonists by default, but antag selection is decided first. For instance, you can set Security Officer to High without affecting your chances of becoming an antag -- the game will just select a different role.
 There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!
 On some maps, you can use a machine in the vault to deposit space cash for cargo points or rob cargo blind.
+If you suspect a container to be booby-trapped with a mousetrap-rigged grenade, you can safely dump its contents by clickdragging it onto an open tile or table.
```

Comment:
this is an exploit tbh

Reply:
good luck fixing it

Reply:
If we acknowledge it's an exploit and not a exploit-elevated-into-feature we probably shouldn't put it as a tip regardless of how much of a pain it'll be to fix it.

Reply:
Yeah this is a pretty obvious exploit. Remove this please.